coalib/output/ConsoleInteraction.py

```diff
@@ -766,15 +766,18 @@ def print_bears(bears,
         console_printer.print('No bears to show. Did you forget to install '
                               'the `coala-bears` package? Try `pip3 install '
                               'coala-bears`.')
-        return
+        return None
 
-    for bear, sections in sorted(bears.items(),
-                                 key=lambda bear_tuple: bear_tuple[0].name):
```

Comment:
Moreover a function like `print_bears` shouldn't return any value. Not at all a value like `sorted_bears`. Maybe you can test without returning any value from the function.